schema/bundle.schema.json

```diff
@@ -104,12 +104,16 @@
         },
         "parameters":{
             "description": "Parameters that can be injected into the invocation image",
-            "type": "object",
```

Comment:
was `"type": "object"` purposefully deleted?

Reply:
Nope. That is a mistake. We will revert that bit.